salesagility/SuiteCRM
# Fixed #7027 - Use a maintained library to minify JS

## Description



Part of my ongoing efforts to remove manually included libraries and replace them with composer packages. This PR replaces the outdated [tedious/JShrink](https://github.com/tedious/JShrink) and instead loads an updated version with composer. I've also re-minified the existing JS files using the updated library.

## Motivation and Context

Issue reference: #7027 

## How To Test This

1. Run JS repairs in admin panel.
2. Check that minified JS works as expected.

## Types of changes


### Final checklist
